### ISSUE TITLE
Dynamic Filtering doesn't work in Edge or Internet Explorer

### ISSUE THREAD

**Issue by DarylMcCullough** [visjs/vis-network#959]
The new Dynamic Filtering capability featured on the url

https://visjs.github.io/vis-network/examples/network/data/dynamicFiltering.html

works great in Google Chrome, but does not work in Internet Explorer or Edge.

I would like to request that this feature be extended to work in those browsers, as well.

Are there plans to make it work in those browsers?

**Comment by Thomaash** [comment]
Hi @DarylMcCullough,

the example happens to be using syntax from 2015 (IE didn't get to 2015 yet and AFAIK never will) so the example fails. However the error comes from the HTML, not from the imported library, the filtering itself should work (we polyfill everything on [this list](https://browsersl.ist/?q=%3E+0.1%25+and+not+dead) and IE11 is still there).

Regarding Edge, isn't it just yet another Chrome in disguise? Shouldn't everything that works in Chrome work in Edge too?

BTW the filtering is not new, it has been implemented long before I joined the community.
